rust-runtime/aws-smithy-http/src/property_bag.rs

```diff
@@ -225,22 +243,30 @@ impl From<PropertyBag> for SharedPropertyBag {
 }
 
 #[cfg(test)]
-#[test]
-fn test_extensions() {
-    #[derive(Debug, PartialEq)]
-    struct MyType(i32);
+mod test {
+    use crate::property_bag::PropertyBag;
+
+    #[test]
+    fn test_extensions() {
+        #[derive(Debug, PartialEq)]
+        struct MyType(i32);
 
-    let mut extensions = PropertyBag::new();
+        let mut property_bag = PropertyBag::new();
 
-    extensions.insert(5i32);
-    extensions.insert(MyType(10));
+        property_bag.insert(5i32);
+        property_bag.insert(MyType(10));
 
-    assert_eq!(extensions.get(), Some(&5i32));
-    assert_eq!(extensions.get_mut(), Some(&mut 5i32));
+        assert_eq!(property_bag.get(), Some(&5i32));
+        assert_eq!(property_bag.get_mut(), Some(&mut 5i32));
 
-    assert_eq!(extensions.remove::<i32>(), Some(5i32));
-    assert!(extensions.get::<i32>().is_none());
+        assert_eq!(property_bag.remove::<i32>(), Some(5i32));
+        assert!(property_bag.get::<i32>().is_none());
 
-    assert_eq!(extensions.get::<bool>(), None);
-    assert_eq!(extensions.get(), Some(&MyType(10)));
+        assert_eq!(property_bag.get::<bool>(), None);
+        assert_eq!(property_bag.get(), Some(&MyType(10)));
+        assert_eq!(
+            format!("{:?}", property_bag),
+            r#"PropertyBag { contents: ["aws_smithy_http::property_bag::test::test_extensions::MyType"] }"#
```

Comment:
`std::any::type_name`'s output is essentially unstable, so I'm not sure if we want this test, see https://doc.rust-lang.org/std/any/fn.type_name.html#note.